pkg/cio/io.go

```diff
@@ -286,18 +296,27 @@ func BinaryIO(binary string, args map[string]string) Creator {
 
 // TerminalBinaryIO forwards container STDOUT|STDERR directly to a logging binary
 // It also sets the terminal option to true
-func TerminalBinaryIO(binary string, args map[string]string) Creator {
+func TerminalBinaryIO(binary string, args map[string]string, setStderr ...bool) Creator {
```

Comment:
Why a variadic?

Reply:
Responded in [following comment_r1663531978](https://github.com/containerd/containerd/pull/10412#discussion_r1663531978)